lib/wings/models/file_node.rb

```diff
@@ -5,75 +5,80 @@ class FileNode < ::Valkyrie::Resource
+    #       needed as all the other AF related resources include this.
+    # attribute :member_ids, ::Valkyrie::Types::Set
 
     attribute :file_set_id, ::Valkyrie::Types::ID
+    attribute :file_identifiers, ::Valkyrie::Types::Set
 
-    attribute :label, ::Valkyrie::Types::Set # AF::File metadata_node label
-    attribute :mime_type, ::Valkyrie::Types::Set # AF::File metadata_node mime_type
-    attribute :format_label, ::Valkyrie::Types::Set # AF::File metadata_node format_label e.g. "JPEG Image"
-    attribute :height, ::Valkyrie::Types::Set # AF::File metadata_node height
-    attribute :width, ::Valkyrie::Types::Set # AF::File metadata_node width
-    attribute :checksum, ::Valkyrie::Types::Set # AF::File metadata_node original_checksum
-    attribute :size, ::Valkyrie::Types::Set # AF::File metadata_node file_size
-    attribute :original_filename, ::Valkyrie::Types::Set # AF::File metadata_node file_name
-    attribute :file_identifiers, ::Valkyrie::Types::Set # AF::File metadata_node
+    # all remaining attributes are on AF::File metadata_node unless otherwise noted
+    attribute :label, ::Valkyrie::Types::Set
+    attribute :original_filename, ::Valkyrie::Types::Set
+    attribute :mime_type, ::Valkyrie::Types::Set
     attribute :use, ::Valkyrie::Types::Set # AF::File type
-    attribute :member_ids, ::Valkyrie::Types::Set
-
-    # TODO: Determine which of the AF::File metadata_node.attributes should be included here.
-    # {"id"=>
-    #      "http://127.0.0.1:8984/rest/dev/gh/93/gz/48/gh93gz487/files/759c2660-3b62-41a2-b453-d41366595062",
-    #  "mime_type"=>[],
-    #  "label"=>[],
-    #  "file_name"=>[],
-    #  "file_size"=>[],
-    #  "date_created"=>[],
-    #  "date_modified"=>[],
-    #  "byte_order"=>[],
-    #  "file_hash"=>[],
-    #  "bit_depth"=>[],
-    #  "channels"=>[],
-    #  "data_format"=>[],
-    #  "frame_rate"=>[],
-    #  "bit_rate"=>[],
-    #  "duration"=>[],
-    #  "sample_rate"=>[],
-    #  "offset"=>[],
-    #  "format_label"=>[],
-    #  "well_formed"=>[],
-    #  "valid"=>[],
-    #  "fits_version"=>[],
-    #  "exif_version"=>[],
-    #  "original_checksum"=>[],
-    #  "file_title"=>[],
-    #  "creator"=>[],
-    #  "page_count"=>[],
-    #  "language"=>[],
-    #  "word_count"=>[],
-    #  "character_count"=>[],
-    #  "line_count"=>[],
-    #  "character_set"=>[],
-    #  "markup_basis"=>[],
-    #  "markup_language"=>[],
-    #  "paragraph_count"=>[],
-    #  "table_count"=>[],
-    #  "graphics_count"=>[],
-    #  "compression"=>[],
-    #  "height"=>[],
-    #  "width"=>[],
-    #  "color_space"=>[],
-    #  "profile_name"=>[],
-    #  "profile_version"=>[],
-    #  "orientation"=>[],
-    #  "color_map"=>[],
-    #  "image_producer"=>[],
-    #  "capture_device"=>[],
-    #  "scanning_software"=>[],
-    #  "gps_timestamp"=>[],
-    #  "latitude"=>[],
-    #  "longitude"=>[],
-    #  "aspect_ratio"=>[]}
+
+    # attributes set by fits
```

Comment:
All remaining changes to this file are adding attribute definitions for attributes set by fits.